client/src/components/map/openlayers/styles.css

```diff
@@ -0,0 +1,3 @@
+.openlayers-map {
```

Comment:
.css must be  in src/style/your_css
!important its bad practice

Reply:
Thank you for hint. Already fixed. Check it, please.